src/duplocloud/client.py

```diff
@@ -33,12 +33,12 @@ def __init__(self,
                query: t.QUERY=None,
                output: t.OUTPUT="json",
                version: t.VERSION=None) -> None:
-    self.host = host
-    self.tenant = tenant
-    self.query = query
-    self.output = output
+    self.host = host.strip()
+    self.tenant = tenant.strip()
+    self.query = query.strip()
+    self.output = output.strip()
     self.timeout = 30
-    self.version = version
+    self.version = version.strip()
```

Comment:
I don't think this needs to be stripped because it's only a `-v` or `--version`.